PG-1527 (part 1): Added base target to aboutBox.htm

tombogle opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/sillsdev/Glyssen/895)
<!-- Reviewable:end -->

tombogle committed:
PG-1527 (part 1): Added base target to aboutBox.htm

andrew-polk reviewed (approved):
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/Glyssen/895)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @tombogle)



<!-- Sent from Reviewable.io -->

tombogle committed:
PG-1527: Updated to latest release of SIL DLLs

This should get links in About Box to open in default system browser and also cause CSS to apply correctly.

andrew-polk reviewed (approved):
> Reviewed 27 of 27 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/Glyssen/895)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @tombogle)



<!-- Sent from Reviewable.io -->
